README.md

```diff
@@ -27,6 +29,10 @@ You will need the following services available:
 * Zookeeper 3.4.6
 * Redis
 
+You will also need:
```

Comment:
You're right, it is - as below, I've moved it section to the manual installation section